pal/inc/pal.h

```diff
@@ -187,36 +209,12 @@ extern "C" {
 
 /******************* Compiler-specific glue *******************************/
 
-#ifndef _MSC_VER
 #define FEATURE_PAL_SXS 1
-#endif // !_MSC_VER
-
-#if defined(_MSC_VER) || defined(__llvm__)
 #define DECLSPEC_ALIGN(x)   __declspec(align(x))
-#else
-#define DECLSPEC_ALIGN(x)
-#endif
-
 #define DECLSPEC_NORETURN   PAL_NORETURN
-
-#ifndef _MSC_VER
 #define __assume(x) (void)0
```

Comment:
I'm trying to understand the changes to this file- is the idea basically that this file should never be built with VC++? Otherwise, for instance, this change breaks prefast, no?